internal-messaging/message/cc/CommandMessage.cc

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include <cstring>
+#include "CommandMessage.h"
+#include <stdexcept>
+
+CommandMessage::CommandMessage()
+:Message()
+{
+    flag = 1;
```

Comment:
Variables such as "flag" should **always** be stored in constant variables, and defined only once.

Reply:
Im not sure what a better way to initialize flag would be. flag is a protected member of Message, not specific to DataMessage. So it cannot be initialized in the DataMessage.h header. I  could make it a parameter of Messages constructor which I am calling, but I am not sure that is better.

Can you suggest a implementation?